### ISSUE TITLE
Touch mode

### ISSUE THREAD

**Issue by Dadibom** [farin/JCloisterZone#154]
A touch friendly interface would be great. An easier way to undo, skip (larger button would be fine), rotate a placed tile and placing meeples.

**Comment by x-iso** [comment]
Agreed, touch mode would require additional confirmation buttons for placing a tile or other actions and making tiles rotate on repeated tile tap. some target pictures also would be necessary to show what areas are available for placement.

**Comment by Dadibom** [comment]
Possible tile positions are already marked!

Date: Fri, 31 Jul 2015 16:29:18 -0700

**Comment by x-iso** [comment]
Those positions are marked by mouse hover, which does not help much when using touch.

**Comment by Dadibom** [comment]
No, there are squares that show where you can place the tile

Date: Sat, 1 Aug 2015 04:09:33 -0700

**Comment by x-iso** [comment]
I see, but I was talking about targets within the tiles (roads, city parts, etc)

**Comment by x-iso** [comment]
any hope for this feature to be added?

**Comment by farin** [comment]
It's currently not a priority and with limited time I can spent with development I wouldn't expect it soon
